### ISSUE TITLE
[Style] Redesign the logo of the project website

### ISSUE THREAD

**Issue by eshita-bandyopadhyay-coder** [Rakesh9100/Eventica#5]
### What's the style idea?

An attractive and appealing Logo can be created for the website , which will enhance it further.

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Eventica/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a KWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Eventica/blob/main/.github/CODE_OF_CONDUCT.md)".

**Comment by eshita-bandyopadhyay-coder** [comment]
DESCRIPTION :  Figma and Canva can be used to create a logo element.

I would like to participate in solving this , please assign me this task.

**Comment by Rakesh9100** [comment]
Go ahead @eshita-bandyopadhyay-coder

**Comment by eshita-bandyopadhyay-coder** [comment]
@Rakesh9100 sir will I first be sharing the logos I make before,  finalizing and adding it for the project ?

**Comment by Rakesh9100** [comment]
Yes @eshita-bandyopadhyay-coder

**Comment by eshita-bandyopadhyay-coder** [comment]
https://www.canva.com/design/DAGZE2wwfmw/LpVOnlBHwnjmgiDiOx187Q/view?utm_content=DAGZE2wwfmw&utm_campaign=designshare&utm_medium=link2&utm_source=uniquelinks&utlId=h9c50786c2f

@Rakesh9100 sir please check the above link and suggest the improvisations

**Comment by abhirajadhikary06** [comment]
@eshita-bandyopadhyay-coder Your logo design looks great. But if you could give it a profession touch I guess it would be much better. For reference you can go through other event booking platforms like **Eventbrite**, **BookMyShow**, **Eventify**.

And if @Rakesh9100 approves your logo then you are good to go.
@Rakesh in my opinion if this project is worked upno maybe it could become a sole open source Event booking platform in future.

Thanks!

**Comment by eshita-bandyopadhyay-coder** [comment]
@abhirajadhikary06 thank you so much for the advice , i will work more upon it.

**Comment by Rakesh9100** [comment]
Logo seems good but instead of the horizontal design, we can have circular and some good logo inside that which will look good @eshita-bandyopadhyay-coder 

Good suggestion, will look into further as project is already kept for open source@abhirajadhikary06

**Comment by Rakesh9100** [comment]
The logo seems good but instead of the horizontal design, we can have a circular and some good logo inside that will look good @eshita-bandyopadhyay-coder 

Good suggestion, will look into it further as the project is already kept for open source, currently it is for Bangalore as I reside here but can be improved much more in future @abhirajadhikary06

**Comment by eshita-bandyopadhyay-coder** [comment]
correct sir @Rakesh9100 , will present my work according to the suggestion given by you.

**Comment by abhirajadhikary06** [comment]
> The logo seems good but instead of the horizontal design, we can have a circular and some good logo inside that will look good @eshita-bandyopadhyay-coder
> 
> Good suggestion, will look into it further as the project is already kept for open source, currently it is for Bangalore as I reside here but can be improved much more in future @abhirajadhikary06

Thanks for considering my suggestion. We can develop it to an open source event booking just for events in Bangalore only (niche area).

**Comment by Rakesh9100** [comment]
Any update on this @eshita-bandyopadhyay-coder

**Comment by eshita-bandyopadhyay-coder** [comment]
Yes sir , was working on the previous issue, will now work on this and update
.

**Comment by eshita-bandyopadhyay-coder** [comment]
https://www.canva.com/design/DAGaCpPkpew/MailbYE28OaTose-BIDL6Q/edit?utm_content=DAGaCpPkpew&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

sir here goes the logo kindly let me know for any improvizations.

**Comment by eshita-bandyopadhyay-coder** [comment]
https://www.canva.com/design/DAGaCpPkpew/MailbYE28OaTose-BIDL6Q/edit?utm_content=DAGaCpPkpew&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

sir here goes the logo

**Comment by abhirajadhikary06** [comment]
@eshita-bandyopadhyay-coder This is good but will not work as this logo will increase the navbar width.

**Comment by Rakesh9100** [comment]
Logo seems good but will it maintain the size consistency in navbar and visible clearly without any blur @eshita-bandyopadhyay-coder

**Comment by eshita-bandyopadhyay-coder** [comment]
sir i will improvise upon it and update my another work.
thank you @Rakesh9100 sir and @abhirajadhikary06 sir

**Comment by eshita-bandyopadhyay-coder** [comment]
@Rakesh9100 sir , i will get back on the improvization after a week , due to some unavoidable circumstances.

**Comment by eshita-bandyopadhyay-coder** [comment]
https://www.canva.com/design/DAGbDl47WEw/wgwmTWlDuNQF9KGcerQKiw/edit?utm_content=DAGbDl47WEw&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton

sir please review... this

**Comment by abhirajadhikary06** [comment]
> https://www.canva.com/design/DAGbDl47WEw/wgwmTWlDuNQF9KGcerQKiw/edit?utm_content=DAGbDl47WEw&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton
> 
> sir please review... this

Please make the link public.

**Comment by Rakesh9100** [comment]
Please make the link shareable and accessible @eshita-bandyopadhyay-coder

**Comment by eshita-bandyopadhyay-coder** [comment]
![image](https://github.com/user-attachments/assets/542eb2a1-3f04-4743-90c4-537687ecc8f5)

please check now sir

**Comment by Rakesh9100** [comment]
Logo has the clarity, but design is too simple, can you try using some colors and enhance the design instead of keeping black and white @eshita-bandyopadhyay-coder

**Comment by eshita-bandyopadhyay-coder** [comment]
sure sir @Rakesh9100

**Comment by Rakesh9100** [comment]
Any update @eshita-bandyopadhyay-coder

**Comment by eshita-bandyopadhyay-coder** [comment]
Hello @Rakesh9100 sir , i just looked upon the live project and I suggest the logo which you are having on thr project currently is looking good. 
i suggest that changes are not that much required.

**Comment by eshita-bandyopadhyay-coder** [comment]
@Rakesh9100 sir i want to raise a PR of this.
 last design .
